arm-analysisservices/2016-05-16/swagger/analysisservices.json

```diff
@@ -519,8 +519,9 @@
         "asAdministrators": {
           "$ref": "#/definitions/ServerAdministrators"
         },
-        "backupConfiguration": {
```

Comment:
@vishrutshah could you please take a look at the pull request?
https://github.com/Azure/azure-rest-api-specs/pull/1296